storops/unity/resource/host.py

```diff
@@ -120,7 +196,52 @@ def get_resource_class(cls):
 
 
 class UnityHostInitiator(UnityResource):
-    pass
+    @classmethod
+    def create(cls, cli, name, host, type, is_ignored=None,
```

Comment:
Could we save the parameter type?  I think `name` here is a wwn or an iqn.  And you could conclude the correct `type` of this initiator by analyzing this `name`.